powersimdata/network/helpers.py

```diff
@@ -60,7 +60,12 @@ def interconnect_to_name(interconnect, model="hifld"):
     :param str model: the grid model.
     :return: (*str*): name of grid model.
     """
-    return "_".join(sorted(check_and_format_interconnect(interconnect, model=model)))
+    return "_".join(
+        sorted(
+            i.replace(" ", "")
+            for i in check_and_format_interconnect(interconnect, model=model)
+        )
+    )
```

Comment:
I think [l.40](https://github.com/Breakthrough-Energy/PowerSimData/blob/f652b5b67743003af20e597ee6109c5688570f8f/powersimdata/network/helpers.py#L40) in `check_and_format_interconnect` will take care of the spaces, no?

Reply:
Yeah. I forgot to remove it in #631 after adding it in the `check_and_format_interconnect` function, see [comment](https://github.com/Breakthrough-Energy/PowerSimData/pull/631#discussion_r879899038)

Reply:
Yeah. I forgot to remove it in #631 after adding it in the `check_and_format_interconnect` function, see [comment](https://github.com/Breakthrough-Energy/PowerSimData/pull/631#discussion_r879899038)

Reply:
Done